.github/workflows/dev-be-ci.yaml

```diff
@@ -32,6 +30,12 @@ jobs:
         with:
           go-version-file: 'go.mod'
           cache: true
+          
+      - name: Setup Go modules
```

Comment:
please leave a comment explaining why we need this step

Reply:
done